lib/lotus/commands/generate/action.rb

```diff
@@ -92,7 +92,17 @@ def skip_view?
         # @since x.x.x
         # @api private
         def http_method
-          options.fetch(:method, DEFAULT_HTTP_METHOD).downcase
+          options.fetch(:method, resourceful_http_method).downcase
+        end
+
+        # @since x.x.x
+        # @api private
+        def resourceful_http_method
+          {
```

Comment:
@cllns Can you please extract this mapping into a constant as well? Thank you very much.